resources/bin/ballerina.bat

```diff
@@ -35,17 +35,11 @@ if "%1" == "update" set update=true
 if "%1" == "build" set build=true
 SetLocal EnableDelayedExpansion
 
-if exist %CURRENT_PATH%..\dependencies\jdk8u202-b08-jre (
-   set JAVA_CMD=%CURRENT_PATH%..\dependencies\jdk8u202-b08-jre\bin\java
+if exist %CURRENT_PATH%..\dependencies\jdk-11.0.8+10-jre (
+   set JAVA_CMD=%CURRENT_PATH%..\dependencies\jdk-11.0.8+10-jre\bin\java
 ) else (
     if not exist "%JAVA_HOME%" (
-        echo Compatible JRE 8 not found. Please follow the instructions in ^<BALLERINA_HOME^>\INSTALL.txt to install and setup Ballerina.
-        exit /b
-    )
-
-    "%JAVA_HOME%\bin\java" -version 2>&1 | findstr /r "1.8">NUL
-    if errorlevel 1 (
-        echo Compatible JRE 8 not found. Please follow the instructions in ^<BALLERINA_HOME^>\INSTALL.txt to install and setup Ballerina.
+        echo Compatible JRE not found. Please follow the instructions in ^<BALLERINA_HOME^>\INSTALL.txt to install and setup Ballerina.
```

Comment:
IMO it would be better if we can mention the supported JRE version in the error description, just like we used to. WDYT?

Reply:
Update tool should work with any JRE so that's the reason this is not been checked in this repo.  

With jdk 11 migration for ballerina-lang we need to revisit this check again everywhere. There we should not exit with error as we did before for jdk 1.8. We are expecting it should work with newer jdks. Probably we might need to give a warning or let it execute without any message. 

We will be verifying this by testing once full migration is done. Until that we have removed jdk checks.

Reply:
@keizer619 Ack. Then lets add the related changes in a separate PR after verifying with different JRE versions. +1